besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -0,0 +1,540 @@
+/*
```

Comment:
would be nice if we could create constants for the "magic" numbers, but if we cannot do that we should provide some kind of clue where these numbers come from ...